src/components/MagicCodeInput.js

```diff
@@ -323,7 +320,7 @@ function MagicCodeInput(props) {
                         >
                             <Text style={[styles.magicCodeInput, styles.textAlignCenter]}>{decomposeString(props.value, props.maxLength)[index] || ''}</Text>
                         </View>
-                        <View style={[StyleSheet.absoluteFillObject, styles.w100, isMobileSafari ? styles.bgTransparent : styles.opacity0]}>
+                        <View style={[StyleSheet.absoluteFillObject, styles.w100, styles.bgTransparent]}>
```

Comment:
Why are we always passing `styles.bgTransparent` here? Can you add some comment in the code above this to explain what purpose that serves?

Reply:
Please add a comment here.

Reply:
Reading the comment made me wonder why would we need this now in the first place, what does it do?

Reply:
My understanding is that we're basically doing a "hidden input" trick here.

Reply:
I'm just curious. Now we do the trick with styles and props on the `TextInput` itself.  So I don't understand why would need this, let me know if am missing anything thanks!